code/datums/components/fertile_egg.dm

```diff
@@ -0,0 +1,67 @@
+/**
+ * # A fertile egg component!
```

Comment:
fair warning I myself stopped using `#` in these and instead use `###` because it is huge when you look at the documentation

# it looks like THIS in vscode